app/models/group.js

```diff
@@ -59,12 +59,13 @@ groupSchema.pre('save', function (next) {
   if (this.inLDAP) {
     return next();
   }
+  var that = this;
```

Comment:
why do you have this -- I REALLLLLLLLY hate this -- it's very unclear There has to be a cleaner way